patch-package
--- a/node_modules/harfbuzzjs/dist/index.mjs
+++ b/node_modules/harfbuzzjs/dist/index.mjs
@@ -2319,7 +2319,7 @@ function otTagToLanguage(tag) {
 //#endregion
 //#region src/index.ts
-init(await createHarfBuzz({ locateFile: () => '/harfbuzz.wasm' }));
+init(await createHarfBuzz());
 //#endregion
 export { AxisFlags, Blob, Buffer, BufferContentType, BufferFlag, BufferSerializeFlag, BufferSerializeFormat, ClusterLevel, ColorPaletteFlags, Direction, DrawFuncs, Face, Feature, Font, FontFuncs, GlyphClass, GlyphFlag, MetricsTag, PaintCompositeMode, PaintExtend, PaintFuncs, TracePhase, Variation, otTagToLanguage, otTagToScript, shape, shapeWithTrace, version, versionString };
